src/JuliaInterpreter.jl

```diff
@@ -115,6 +115,50 @@ end
 JuliaStackFrame(frame::JuliaStackFrame, pc::JuliaProgramCounter; kwargs...) =
     JuliaStackFrame(frame.code, frame, pc; kwargs...)
 
+"""
+`LocalVariable` represents a local variable inside a frame.
+By calling the function `locals`[@ref] on a `JuliaStackFrame`[@ref] a
+`Vector` of `LocalVariable`'s is returned.
+
+Important fields:
+- `value::Any`: the value of the local variable
+- `name::Symbol`: the name of the variable as given in the source code
+- `isparam::Bool`: if the variable is a type parameter, for example `T` in `f(x::T) where {T} = x` .
+"""
+struct LocalVariable
```

Comment:
Perhaps this should just be called `Variable`, is there anything specific about it being `local`. If we want to show all globals, we would presumably use the same struct.